src/gui/widgets/overlay/subtitlewidget.cpp

```diff
@@ -467,6 +478,14 @@ void SubtitleWidget::adjustVisibility()
 
 void SubtitleWidget::positionChanged(const double value)
 {
+    if (m_settings.pauseOnSubtitleEnd
+        && !pausedForCurrentSubtitle
+        && (m_subtitle.endTime - value <= 0))
+    {
+         GlobalMediator::getGlobalMediator()->controlsPause();
```

Comment:
Use `GlobalMediator::getGlobalMediator()->getPlayerAdapter()->pause();` because this isn't sent from PlayerControls.
Also when sending signals be sure to use `Q_EMIT` even though it isn't necessary.